client/pages/recycle-and-reuse.js

```diff
@@ -54,28 +54,30 @@ function RecycleAndReuse({ options }) {
           // variant="enclosed-colored"
           id="tabs-3--tab--1"
           isLazy
+          index={tabIndex}
+          onChange={(index) => setTabIndex(index)}
         >
           <TabList>
-            <Tab>
+            <Tab isDisabled={tabIndex !== 0}>
               <AddIcon />
               Add Item
             </Tab>
-            <Tab isDisabled={hasNoItems(items)}>
+            <Tab isDisabled={tabIndex !== 1}>
               <InfoOutlineIcon />
               Item List
             </Tab>
-            <Tab isDisabled={hasNoItems(items)}>
+            <Tab isDisabled={tabIndex !== 2}>
               <DeleteIcon />
               Dispose Items
             </Tab>
           </TabList>
           <TabPanels>
             <TabPanel>
-              <Additem />
-              <MultiSelect options={options} setInput={setItems} />
+              <Additem navigateToVerifyItem={() => setTabIndex(1)} />
+              <MultiSelect data={data} setItems={setItems} navigateToVerifyItem={() => setTabIndex(1)} />
             </TabPanel>
             <TabPanel>
-              <p>Item List goes here!</p>
+              <VerifyItem items={items} setItems={setItems} navigateToTakeAction={() => setTabIndex(2)} />
```

Comment:
Changes here are to enable auto-tab navigation. So now, after we are done with the first tab (e.g. `AddItem`), we will move on to the second tab (`VerifyItem`) straightaway without having to manually click on the tabs (which was unintuitive to me).